plugins/modules/database/saphana/hana_query.py

```diff
@@ -46,13 +47,13 @@
         type: bool
         default: true
     host:
-        description: The Host IP address. The port can be defined as well.
+        description: The Host IP address. The port can be defined as well. HOW?
```

Comment:
```suggestion
        description: The Host IP address. The port can be defined as well. HOW?
```
Would be great to add an example (at least here and maybe somewhere in EXAMPLES)